### DISCUSSION TITLE
All panorama photos from Galaxy S21 Ultra are considered broken during indexing

### DISCUSSION THREAD

**Discussion by maximebel** [photoprism/photoprism#1945] [General]
**What does not work as expected?**

All my panorama photos from my Galaxy S21 Ultra are considered broken and put in the Hidden section.

There is a an exclamation mark with EOF

**How can we reproduce it?**

If reporting an Import, Indexing, or Performance issue, please include the number and type of pictures in your library,
as well as any configuration options you have changed e.g. for thumbnail quality.

Just import any panorama picture saved by the S21 Ultra

My settings are the default ones, nothing has been changed

**What behavior do you expect?**

Proper import :)

**What could be the cause of your problem?**

I tried to open the panorama photos in many photo softwares and none of them complained.
The only thing I can think of is that they are big jpeg files.

**Can you provide us with example files for testing, error logs, or screenshots?**

File sample will be sent to samples@photoprism.app

**Which software versions do you use?**

- PhotoPrism Architecture & Build Number (AMD64, ARM64, ARMv7,...)  Docker image from Jan 7th 2022
- Database Type & Version (MariaDB, MySQL, SQLite,...) : Mariadb as a docker image 10.6
- Operating System Types & Versions (Linux, Windows, Android,...) NAS synology DSM 7 (18Gb memory so unlikely to be the issue)
- Browser Types & Versions (Firefox, Chrome, Safari on iPhone,...) Chrome but irrevelant for the issue
- Browser Plugins & Ad Blockers (if any) irrevelant for the issue

**On what kind of device is PhotoPrism installed?**

This is especially important if you are reporting a Performance, Import, or Indexing problem. You can skip this if you are reporting a problem found on our public demo, or if it is a completely unrelated issue, such as a broken page layout.

- Device / Processor Type (Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...) DS220+ => Intel Celeron J4025
- Physical Memory & Swap Space (in GB) 18Gb
- Storage Type (HDD, SSD, RAID, USB, Network Storage,...)  RAID & Btrfs
- anything else that might be helpful

IMPORTANT: Always provide device, memory, and storage details when you have a Backend, Performance, Import, or Indexing issue.

**Do you use a Reverse Proxy, Firewall, VPN, or CDN?**

N/A

**Comment by graciousgrey** [comment]
This is not a bug on our side.

Samsung is creating panorama files with not standard conform metadata, it's a known issue.

This thread helped a user with the same problem fixing the files:
- https://exiftool.org/forum/index.php?topic=12349.0

**Comment by kavemang** [comment]
@graciousgrey I'm experiencing this with 360 panoramas from s22 ultra

**Reply by lastzero** [reply]
It's because these are currently not supported, including any other special formats individual vendors may come up with to break general compatibility.
